portality/view/admin.py

```diff
@@ -435,7 +435,12 @@ def application_quick_reject(application_id):
     sent = False
     send_report = []
     try:
-        send_report = emails.send_publisher_reject_email(application, note=reason, update_request=update_request)
+        if self.source.application_status != self.target.application_status:
+            eventsSvc.trigger(models.Event(constants.EVENT_APPLICATION_STATUS, account.id, {
+                "application": self.target.data,
+                "old_status": self.source.application_status,
+                "new_status": self.target.application_status
```

Comment:
self.target and self.source don't exist here, right?  You'll need to pass `application.data` and then get the before application_status from the application when you first load it.  Then the new status is obviously rejected, which we can take from the `constants` module